crates/config/src/lib.rs

```diff
@@ -886,7 +886,7 @@ impl Config {
                         if self.offline {
                             return Err(SolcError::msg(format!(
                                 "can't install missing solc {version} in offline mode"
-                            )))
```

Comment:
Nit: please undo the inserted `;`

See: https://github.com/foundry-rs/foundry/blob/master/CONTRIBUTING.md#resolving-an-issue for the linter settings (we are using `nightly` fmt and `nightly` clippy)